src/charts/backgroundBars.js

```diff
@@ -14,31 +14,32 @@ export default function(ctx) {
 
 	var data = [];
 
-	var timeout;
+	var timeout, barChart;
 
-	for (var i = count - 1; i >= 0; i--) {
+	for (let i = count - 1; i >= 0; i--) {
```

Comment:
I would keep `var` here

Reply:
That was changed, because there is another `let i` in line 51 and I didn't think it should use upper scope `i` there.
Maybe its better to rename the variable instead.

Reply:
I didn't noticed that there is already a `let` at line 51 so I guess it doesn't really matter.